main.cpp

```diff
@@ -0,0 +1,55 @@
+#include <iostream>
+#include <iomanip>
+
+// Assignment one stock pricing and profit
+
+int main() {
+    // Declare Constants
+    int SHARES_BOUGHT = 1000;
+    int SHARES_SOLD = 1000;
+    double COMMISSION_RATE = 0.02;
+    double PURCHASE_PRICE =45.50;
+    double SALE_PRICE = 56.90;
```

Comment:
Perfect all uppercase snake case for your constants. But you need to also provide the `const` keywork to make them immutable.
```
const int SHARES_BOUGHT = 1000;
```